src/core_modules/capture-core/components/Pages/common/TEIRelationshipsWidget/RegisterTei/RegistrationSection/RegUnitSelector/RegUnitSelector.component.tsx

```diff
@@ -1,26 +1,21 @@
-// @flow
-/* eslint-disable react/no-multi-comp */
-import * as React from 'react';
+import React from 'react';
 import i18n from '@dhis2/d2-i18n';
-import { withStyles } from '@material-ui/core/styles';
+import { withStyles, type WithStyles, type Theme } from '@material-ui/core/styles';
 import { ComposedRegUnitSelector } from './ComposedRegUnitSelector.component';
 import { getProgramFromProgramIdThrowIfNotFound } from '../../../../../../../metaData';
+import type { RegUnitSelectorProps } from './RegUnitSelector.types';
 
 const getStyles = (theme: Theme) => ({
     label: {
         paddingTop: '10px',
-        [theme.breakpoints.down(523)]: {
+        color: theme.palette.text.primary,
+        '@media (max-width: 523px)': {
```

Comment:
Remove color and revert the breakpoint to the following: `[theme.breakpoints.down(523)]: {`